README.md

```diff
@@ -1,6 +1,11 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+<<<<<<< HEAD
```

Comment:
@sanya-kenneth  It seems changes were not applied when we were doing the rebase. Remove from line 2 - 5 plus line 8 and leave the rest. Then amend the commit and everything will be fine.